test/acceptance/todo-cli-test.js

```diff
@@ -89,7 +83,7 @@ describe('todo usage', () => {
       expect(result.stdout).toBeTruthy();
     });
 
-    it('errors when config.pending and `.lint-todo` dir coexist', async function () {
+    it('errors when config.pending is present', async function () {
```

Comment:
I'm not sure this test is useful anymore. Perhaps it should be deleted.

There's already a separate test for the `Unknown top-level configuration property` exception.

Reply:
Yep, will remove with the extra validation code.